### ISSUE TITLE
Unable to specify LD_LIBRARY_PATH

### ISSUE THREAD

**Issue by bill-kidwell** [cloudfoundry/nodejs-buildpack#127]
cf.exe version 6.33.1+c77e55743.2017-12-15

nodejs_Buildpack 1.5.32

We have a node.js application that wraps our C++ code with a native node module.  In order for this to run correctly, we need to set the LD_LIBRARY_PATH.  However, we cannot seem to get this to work.  We have tried setting the environment variable, but from looking at the buildpack code, it appears that it overwrites any values. 

We expected a way to modify the LD_LIBRARY_PATH

Instead, the application fails with a message

2018-02-15T16:30:35.53-0500 [APP/PROC/WEB/0] OUT error loading library
2018-02-15T16:30:35.53-0500 [APP/PROC/WEB/0] OUT libicudata.so.58: cannot open shared object file: No such file or directory

The only way we were able to get this to work was to change LD_LIBRARY_PATH in a custom buildpack.

**Comment by dgodd** [comment]
@bill-kidwell The installation of nodejs needs to set LD_LIBRARY_PATH for node to work. You should be able to add yours as an additional element of the path, how were you trying to set the path? 

Also you may wish to use a supply buildpack to provide `libicudata.so.58` so that it is easier to mainatin seperate to the app.

If you wish to try the supply buildpack path, we have an experimental tool at https://github.com/cloudfoundry/bp-supply/tree/develop.

**Comment by bill-kidwell** [comment]
Thanks for the quick response @dgodd ,  we had attempted to set the LD_LIBRARY_PATH environment variable through the cf cli and in the manifest.   We finally got it to work with a statement like the following:

cf push -c "export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/vcap/app/_ourpath_;node ourscript.js"

Ideas on how we can get the same results in the manifest.yml?

We will check out the supply buildpack.  We knew we needed to separate it,  but we included it in the app in the short term as an incremental step.
